packages/indexer-common/src/indexer-management/models/poi-dispute.ts

```diff
@@ -161,13 +161,13 @@ export const definePOIDisputeModels = (sequelize: Sequelize): POIDisputeModels =
         allowNull: true,
         validate: {
           // eslint-disable-next-line @typescript-eslint/no-explicit-any
-          isHex: (value: any) => {
-            if (typeof value !== 'string') {
-              throw new Error('Allocation POI must be a string')
+          isHexOrNull: (value: any) => {
```

Comment:
validation error if only allow string, though line161 specified that it should be okay... Not sure what is this